package.json

```diff
@@ -31,6 +31,10 @@
     "@commitlint/cli": "^8.0.0",
     "@commitlint/config-conventional": "^7.5.0",
     "@commitlint/config-lerna-scopes": "^7.5.1",
+    "@fortawesome/fontawesome-svg-core": "^1.2.29",
```

Comment:
afaik you don't have to add dependencies here since you are already adding them to a specific package

Reply:
Root is devDependency and here is dependency because we deliver SVGs and fontAwesome render component

Reply:
Removed from main package :)